lib/module.d.ts

```diff
@@ -0,0 +1,6 @@
+export class audio
```

Comment:
nit: classes should be PascalCase in js
```suggestion
export class Audio
```